### ISSUE TITLE
Should xgit remote include RemoteUrl?

### ISSUE THREAD

**Issue by muschellij2** [r-lib/sessioninfo#49]
I use  `sessioninfo::session_info()` for a reproducible list of environmentfs and sometimes have to use `remotes::install_git` (SSH auth).  For example, see below.  The question/bug is whether `xgit` should include Remote URL?

Intended output line for `nsrr` would be:
```
xgit (git@github.com:muschellij2/nsrr@c8feb6a)
```

``` r
library(remotes)
remove.packages("nsrr")
#> Removing package from '/home/jupyter/.R/library'
#> (as 'lib' is unspecified)
remotes::install_git("git@github.com:muschellij2/nsrr", 
                     git = "external", upgrade = FALSE)
#> Downloading git repo git@github.com:muschellij2/nsrr
#> '/usr/bin/git' clone --depth 1 --no-hardlinks git@github.com:muschellij2/nsrr /tmp/RtmpjcPoMa/file606918f11ace
#>      checking for file ‘/tmp/RtmpjcPoMa/file606918f11ace/DESCRIPTION’ ...  ✓  checking for file ‘/tmp/RtmpjcPoMa/file606918f11ace/DESCRIPTION’ (339ms)
#>   ─  preparing ‘nsrr’:
#>    checking DESCRIPTION meta-information ...  ✓  checking DESCRIPTION meta-information
#>   ─  checking for LF line-endings in source and make files and shell scripts
#>   ─  checking for empty or unneeded directories
#>   ─  building ‘nsrr_0.2.1.tar.gz’
#>      
#> 
#> Installing package into '/home/jupyter/.R/library'
#> (as 'lib' is unspecified)
readLines(system.file("DESCRIPTION", package = "nsrr"))
#>  [1] "Type: Package"                                                              
#>  [2] "Package: nsrr"                                                              
#>  [3] "Title: Interface to National Sleep Research Resource"                       
#>  [4] "Version: 0.2.1"                                                             
#>  [5] "Authors@R: "                                                                
#>  [6] "    person(given = \"John\","                                               
#>  [7] "           family = \"Muschelli\","                                         
#>  [8] "           role = c(\"aut\", \"cre\"),"                                     
#>  [9] "           email = \"muschellij2@gmail.com\","                              
#> [10] "           comment = c(ORCID = \"0000-0001-6469-1750\"))"                   
#> [11] "Description: Allows users to access data from the National Sleep"           
#> [12] "    Research Resource ('NSRR') <https://sleepdata.org/>."                   
#> [13] "License: GPL-3"                                                             
#> [14] "Imports: httr, jsonlite, tools, digest, stats, utils"                       
#> [15] "Suggests: covr, DT, knitr, rmarkdown, testthat, xml2, rvest, readr"         
#> [16] "Encoding: UTF-8"                                                            
#> [17] "VignetteBuilder: knitr"                                                     
#> [18] "URL: https://github.com/muschellij2/nsrr"                                   
#> [19] "BugReports: https://github.com/muschellij2/nsrr/issues"                     
#> [20] "RoxygenNote: 7.1.1"                                                         
#> [21] "RemoteType: xgit"                                                           
#> [22] "RemoteUrl: git@github.com:muschellij2/nsrr"                                 
#> [23] "RemoteSha: c8feb6ad1d33aaddc9cec95df3a2c362e2772dac"                        
#> [24] "NeedsCompilation: no"                                                       
#> [25] "Packaged: 2021-10-20 01:58:49 UTC; jupyter"                                 
#> [26] "Author: John Muschelli [aut, cre] (<https://orcid.org/0000-0001-6469-1750>)"
#> [27] "Maintainer: John Muschelli <muschellij2@gmail.com>"                         
#> [28] "Built: R 4.1.0; ; 2021-10-20 01:58:49 UTC; unix"
sessioninfo::session_info("nsrr")
#> ─ Session info ───────────────────────────────────────────────────────────────
#>  setting  value                       
#>  version  R version 4.1.0 (2021-05-18)
#>  os       Debian GNU/Linux 10 (buster)
#>  system   x86_64, linux-gnu           
#>  ui       X11                         
#>  language (EN)                        
#>  collate  C.UTF-8                     
#>  ctype    C.UTF-8                     
#>  tz       Etc/UTC                     
#>  date     2021-10-20                  
#> 
#> ─ Packages ───────────────────────────────────────────────────────────────────
#>  package  * version    date       lib source         
#>  curl       4.3.2      2021-06-23 [1] CRAN (R 4.1.0) 
#>  digest     0.6.27     2020-10-24 [2] CRAN (R 4.1.0) 
#>  httr       1.4.2.9000 2021-08-31 [1] local          
#>  jsonlite   1.7.2      2020-12-09 [2] CRAN (R 4.1.0) 
#>  mime       0.12       2021-09-28 [1] CRAN (R 4.1.0) 
#>  nsrr       0.2.1      2021-10-20 [1] xgit (@c8feb6a)
#>  openssl    1.4.5      2021-09-02 [1] CRAN (R 4.1.0) 
#>  R6         2.5.1      2021-08-19 [1] CRAN (R 4.1.0) 
#> 
#> [1] /home/jupyter/.R/library
#> [2] /usr/local/lib/R/site-library
#> [3] /usr/lib/R/site-library
#> [4] /usr/lib/R/library
```

<sup>Created on 2021-10-20 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

<details style="margin-bottom:10px;">
<summary>
Session info
</summary>

``` r
sessioninfo::session_info()
#> ─ Session info ───────────────────────────────────────────────────────────────
#>  setting  value                       
#>  version  R version 4.1.0 (2021-05-18)
#>  os       Debian GNU/Linux 10 (buster)
#>  system   x86_64, linux-gnu           
#>  ui       X11                         
#>  language (EN)                        
#>  collate  C.UTF-8                     
#>  ctype    C.UTF-8                     
#>  tz       Etc/UTC                     
#>  date     2021-10-20                  
#> 
#> ─ Packages ───────────────────────────────────────────────────────────────────
#>  package     * version    date       lib source                        
#>  callr         3.7.0      2021-04-20 [2] CRAN (R 4.1.0)                
#>  cli           3.0.1      2021-07-17 [1] CRAN (R 4.1.0)                
#>  crayon        1.4.1      2021-02-08 [2] CRAN (R 4.1.0)                
#>  digest        0.6.27     2020-10-24 [2] CRAN (R 4.1.0)                
#>  evaluate      0.14       2019-05-28 [2] CRAN (R 4.1.0)                
#>  fs            1.5.0      2020-07-31 [2] CRAN (R 4.1.0)                
#>  glue          1.4.2      2020-08-27 [2] CRAN (R 4.1.0)                
#>  highr         0.9        2021-04-16 [2] CRAN (R 4.1.0)                
#>  htmltools     0.5.1.1    2021-01-22 [2] CRAN (R 4.1.0)                
#>  knitr         1.36       2021-09-29 [1] CRAN (R 4.1.0)                
#>  magrittr      2.0.1      2020-11-17 [2] CRAN (R 4.1.0)                
#>  pkgbuild      1.2.0      2020-12-15 [2] CRAN (R 4.1.0)                
#>  prettyunits   1.1.1      2020-01-24 [2] CRAN (R 4.1.0)                
#>  processx      3.5.2      2021-04-30 [2] CRAN (R 4.1.0)                
#>  ps            1.6.0      2021-02-28 [2] CRAN (R 4.1.0)                
#>  R6            2.5.1      2021-08-19 [1] CRAN (R 4.1.0)                
#>  remotes     * 2.4.1.9000 2021-10-20 [1] Github (r-lib/remotes@eb15a1e)
#>  reprex        2.0.0      2021-04-02 [2] CRAN (R 4.1.0)                
#>  rlang         0.4.11     2021-04-30 [2] CRAN (R 4.1.0)                
#>  rmarkdown     2.9        2021-06-15 [2] CRAN (R 4.1.0)                
#>  rprojroot     2.0.2      2020-11-15 [2] CRAN (R 4.1.0)                
#>  rstudioapi    0.13       2020-11-12 [2] CRAN (R 4.1.0)                
#>  sessioninfo   1.1.1      2018-11-05 [2] CRAN (R 4.1.0)                
#>  stringi       1.7.5      2021-10-04 [1] CRAN (R 4.1.0)                
#>  stringr       1.4.0      2019-02-10 [2] CRAN (R 4.1.0)                
#>  withr         2.4.2      2021-04-18 [2] CRAN (R 4.1.0)                
#>  xfun          0.26       2021-09-14 [1] CRAN (R 4.1.0)                
#>  yaml          2.2.1      2020-02-01 [2] CRAN (R 4.1.0)                
#> 
#> [1] /home/jupyter/.R/library
#> [2] /usr/local/lib/R/site-library
#> [3] /usr/lib/R/site-library
#> [4] /usr/lib/R/library
```

</details>

**Comment by gaborcsardi** [comment]
Right, sorry. What will it say after #50?

**Comment by gaborcsardi** [comment]
Closed by #50.
